package.json

```diff
@@ -9,19 +9,24 @@
     "flow": "flow; test $? -eq 0 -o $? -eq 2",
     "build": "npm run build-proto && browserify client/sync.js | uglifyjs > bundles/bundle.js && cpr client/constants bundles/constants -o",
     "build-proto": "pbjs --target=static-module lib/api.proto > lib/api.proto.js",
+    "prebrowsertest": "optional-dev-dependency",
     "browsertest": "npm run build-proto; npm run start-test; browserify test/client/*.js test/*.js | uglifyjs | tape-run; exitCode=$?; npm run stop-test; exit $exitCode",
+    "prebrowsertest-chrome": "optional-dev-dependency",
     "browsertest-chrome": "npm run build-proto; npm run start-test; browserify test/client/*.js | tape-run --browser chrome; exitCode=$?; npm run stop-test; exit $exitCode",
+    "prebrowsertest-safari": "optional-dev-dependency",
     "browsertest-safari": "npm run build-proto; npm run start-test; browserify test/client/*.js | tape-run --browser safari; exitCode=$?; npm run stop-test; exit $exitCode",
     "check": "npm audit",
     "client": "npm run build && python -m SimpleHTTPServer",
-    "precoverage": "optional-dev-dependency -t istanbul",
+    "precoverage": "optional-dev-dependency",
     "coverage": "NODE_CONFIG_DIR=server/config/ istanbul cover tape test/*.js test/server/**/*.js --report lcovonly -- -R spec",
     "dist": "npm run build && cpr bundles/bundle.js ../browser-laptop/app/extensions/brave/content/scripts/sync.js -o && cpr bundles/constants/ ../browser-laptop/js/constants/sync/ -f '\\w+\\.js' -o",
     "start": "NODE_CONFIG_DIR=server/config/ node server/index.js",
     "start-dev": "NODE_CONFIG_DIR=server/config/ nodemon server/index.js",
     "start-test": "NODE_CONFIG_DIR=server/config/ NODE_ENV=test nohup node server/index.js > /dev/null 2>&1 & echo $! > start-test.pid",
     "stop-test": "kill $(cat start-test.pid)",
+    "pretape": "optional-dev-dependency",
     "tape": "NODE_CONFIG_DIR=server/config/ tape",
+    "pretest": "optional-dev-dependency",
```

Comment:
are `pretape` and `pretest` needed? they use `tape`, not `tape-run` i think

Reply:
Good question - may not be needed 😄 Let me give that a go manually

Reply:
nope- doesn't look it! amending commit here...